changesets2CSV/changesets2CSV

```diff
@@ -177,6 +188,10 @@ def changeset_csv(output_file, changesets, name=None, summary=False):
                     if child.get('k') == 'comment':
                         changeset['Comment'] = child.get('v').encode('utf-8')
 
+                changesetInformation = count_new_modified_deleted(changeset['URL'])
```

Comment:
I'm curious how this is effecting performance. Not blocking but wondering if it could be more efficient.

The `[asyncio](https://docs.python.org/3/library/asyncio.html)` module may be useful if we're seeing a major efficiency decrease

Reply:
It does a call per changeset, so it is going to be slower. I could add an argument to use it (i.e., it isn't used by default).

As far as performance goes, I don't want to be causing issues for other OSM users (see https://operations.osmfoundation.org/policies/api/ for why), so I don't think we want to be running download processes asynchronously.

I would rather use python [requests](https://3.python-requests.org/) with [CacheControl](https://3.python-requests.org/community/recommended/#cachecontrol), but that won't affect the initial run, only secondary runs.

Reply:
> I could add an argument to use it (i.e., it isn't used by default).

I don't think this should be a flag. No worries

> As far as performance goes, I don't want to be causing issues for other OSM users (see https://operations.osmfoundation.org/policies/api/ for why), so I don't think we want to be running download processes asynchronously.

That makes sense. I was thinking about it wrong but that sounds good. 


> I would rather use python requests with CacheControl

I'm a fan of the requests module. It's muy bien. We can figure out a solution _if_ we think it's an issue.